apps/site/assets/ts/stop/components/amenities/ElevatorsAmenityCard.tsx

```diff
@@ -1,13 +1,77 @@
 import React from "react";
-import AmenityCard from "./AmenityCard";
+import AmenityCard, { AmenityLink, AmenityModal } from "./AmenityCard";
 import { elevatorIcon } from "../../../helpers/icon";
+import { Alert, Facility } from "../../../__v3api";
+import Alerts from "../../../components/Alerts";
+import { hasFacilityAlert } from "../../../models/alert";
 
-const ElevatorsAmenityCard = (): JSX.Element => {
+const ElevatorsAmenityCard = ({
+  stopName,
+  alerts,
+  facilities
+}: {
+  stopName: string;
+  alerts: Alert[];
+  facilities: Facility[] | undefined;
+}): JSX.Element => {
   const icon = (
     <span className="m-stop-page__icon">{elevatorIcon("c-svg__icon")}</span>
   );
 
-  return <AmenityCard headerText="Elevators" icon={icon} />;
+  const hasElevators = facilities ? facilities.length > 0 : false;
+
+  return (
+    <AmenityCard
+      headerText="Elevators"
+      icon={icon}
+      modalContent={
+        hasElevators && (
+          <AmenityModal headerText={`Elevators at ${stopName}`}>
+            <Alerts alerts={alerts} />
+            <h2 className="h3">Eleavator Status</h2>
```

Comment:
This might be a preference but can we use the `<h3>` tag instead of the `h3` class?